rfcs/0175-appeals-council.md

```diff
@@ -0,0 +1,165 @@
+2) We adhere to the principles of *equality* and *objectivity*. Equality refers to the status of each community member, while objectivity pertains to the consistent, unbiased reasoning behind our decisions.
+3) A *concrete reason* is a verifiable state of affairs that can be demonstrated to other community members upon request.
+4) We acknowledge that *concrete reasons* are not self-explanatory, except in extreme cases, and we will provide explanations relating them to the general principles guiding our decisions.
+5) We affirm that moderation actions must be fair and proportionate to the related issue.
+6) We ensure that concrete reasons for permanent bans are available for inspection by long-standing community members.
+7) We recognize that moderation actions should not be based solely on the volume of complaints.
+8) We commit to treating community members with respect and avoiding derogatory language when addressing concerns.
+9) We affirm our role as moderators is to facilitate a respectful community, not to punish members. We will use language that reflects mutual respect.
+10) We prefer inclusion to exclusion, resorting to permanent exclusion only in cases of extreme necessity. We are committed to exploring ways to prevent exclusion and to engaging constructively in efforts to reinstate excluded members.
+11) To reflect a consensus in a diverse community, the moderation team must embody cultural and viewpoint diversity, respecting members with varying societal views, provided they do not undermine equal community membership.
+12) We respect viewpoint diversity in cultural and political matters when selecting moderation team candidates.
+13) We understand that reasonable moderation is based on past events or future expectations, necessitating respect for historical records and transparency in controversial events.
+14) We uphold that true diversity necessitates exceptional tolerance, actively embracing not only the ideas we favor but, also, those we disagree with. This commitment to tolerance is fundamental in fostering a community where diverse viewpoints can coexist and enrich our collective experience.
+
+## Resolution to Adopt the *Memorandum*
+
+- Incorporate the *Memorandum* text into the moderation repository on GitHub, linking it prominently from the Code of Conduct and all relevant public information sources.
+- Implement one-year term limits for moderation team members, with a one-year hiatus before potential return. Apply this rule retroactively.
+- Align the current moderation team composition and the Code of Conduct with the *Memorandum* principles within three months of adopting this document.
+
+## Resolution to Establish an Appeals Council
+
+- Form an *Appeals Council* of three long-standing community members, reflecting cultural and viewpoint diversity.
```

Comment:
On lemmy.world/c/politics I almost always default to 3 day suspensions, for trolls they just get bored when they see they get banned.  And for the majority of kind people that got heated, they see that and don't get upset that it's disproportionate.  Very rarely except for the most blatant trolling (repeated direct insults to other users and slurs).  I generally don't moderate insults directed at moderators.

Reply:
a few bullets down there is a stipulation that says that appeals council members can only overturn moderation decisions exceeding 3 months, to avoid excessive appeals. Essentially only someone who is excluded from the project for an extended period need appeal.